lib/lunchbot_web/templates/admin/select_menu/view.html.heex

```diff
@@ -8,7 +8,7 @@
     <% end %>
 
     <%= label f, :office %>
-    <%= select f, :office, ["Providence": 1, "Boulder": 2], required: true %>
+    <%= select f, :offices, ["Providence": 1, "Boulder": 2], required: true %>
```

Comment:
Is this actually the correct fix here? What is `@changeset` and what attributes does it have defined?

There are also a number of `:office` related form elements around this that are likely broken by this rename.

Is there any way we could test this?

Reply:
And I still think this is the correct fix because when I got rid of the one-to-many association between office and office_lunch_order, keeping `:office` singular allows the page to render correctly.
`@changeset` holds constraints and validations for a struct to ensure data is safe before inserting into the database. The attributes that it holds when I inspected it were:
#Ecto.Changeset<
  action: nil,
  changes: %{},
  errors: [
    day: {"can't be blank", [validation: :required]},
    office_id: {"can't be blank", [validation: :required]},
    menu_id: {"can't be blank", [validation: :required]}
  ],
  data: #Lunchbot.LunchbotData.OfficeLunchOrder<>,
  valid?: false
>
I don't think` @changeset` has to do with what is wrong with the select tag, but I can be wrong. I don't see how they're correlated. And the rename does break the label element (will update the other `:office` to `:offices`) but I'm not sure how to test this other than looking at the HTML and seeing the label 'for' attribute doesn't match the select 'id'.